### PR TITLE
Homework 15 done

### PR DESCRIPTION
**Pull request by MaksServatyak**

### CODE REVIEW

File: src/Homework15/Employee.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package Homework15;
+
+import java.util.Objects;
+
+public class Employee implements Comparable<Employee>{
+        int age;
```
Comment (Volodymyr95) [inline src/Homework15/Employee.java:6]: where access modifiers?

File: src/Homework15/Employee.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package Homework15;
+
+import java.util.Objects;
+
+public class Employee implements Comparable<Employee>{
```
Comment (Volodymyr95) [inline src/Homework15/Employee.java:5]: equals and hash code has a bad implementation

File: src/Homework15/Main.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package Homework15;
+
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.TreeSet;
+
+public class Main {
+    public static void main(String[] args) {
```
Comment (Volodymyr95) [inline src/Homework15/Main.java:8]: You should fill from console

File: src/Homework15/Developer.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package Homework15;
+
+import java.util.Objects;
+
+public class Developer extends Employee {
+    Developer(String name, Integer age) {
```
Comment (Volodymyr95) [inline src/Homework15/Developer.java:6]: where developer fields

File: src/Homework15/Recruiter.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package Homework15;
+
+public class Recruiter extends Employee {
+    Recruiter(String name, Integer age) {
```
Comment (Volodymyr95) [inline src/Homework15/Recruiter.java:4]: where recruiter fields?

File: src/Homework15/Creator.java [hunk 6]
```diff
@@ -0,0 +1,57 @@
+package Homework15;
+
+import java.util.*;
+
+public class Creator {
+    String name;
```
Comment (Volodymyr95) [inline src/Homework15/Creator.java:6]: access modifier?

**Review by natalia-kozak — APPROVED** [review]

**Pull request merged**